luthier/src/main/java/com/yahoo/bard/webservice/application/LuthierBinderFactory.java

```diff
@@ -25,7 +25,9 @@
     protected ConfigurationLoader getConfigurationLoader() {
         LuthierResourceDictionaries resourceDictionaries = new LuthierResourceDictionaries();
         initializeDictionaries(resourceDictionaries);
-        return new LuthierIndustrialPark(resourceDictionaries, getDimensionFactories());
+        LuthierIndustrialPark park = new LuthierIndustrialPark.Builder(resourceDictionaries)
+                .withDimensionFactories(getDimensionFactories()).build();
+        return park;
```

Comment:
No need for the `park` local variable. We can just return `new LuthierIndustrialPark.Builder()...` directly.

Reply:
Fixed.